src/CHANGES.txt

```diff
@@ -7,6 +7,10 @@
 
 RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
 
+  From Daniel Moody:
+    - Made a small change to scons main __init__.py to set the pickling protocal 
+      to choose the latest which fixed 2 failing test varient dir test from Interactive
```

Comment:
You should probably add that this will impact the .sconsign files written by SCons. Though we're not supporting py2 <-> py3 .sconsign compatability, it's worth noting.  If you use 3.0.1 and then use 3.next and then use 3.0.1 it may fail on read..